news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,190 @@
+
+---
+
+While we try to keep watch over all running contests, we might miss some. If you're aware of a mapping contest that should receive more attention, head to [this forum thread](https://osu.ppy.sh/community/forums/topics/1027590) to submit it for inclusion in the next issue of the Mappers' Report!
+
+## <a id="around-community"></a>Around the Community
+
+This section is all about interesting things which have been happening or will happen soon in the mapping and modding community! Should you have/know about something worth including, be sure to let [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323) know about it! For this month, we have updates on the mapping podcast **The osu! Map Cast**, the **Electoz Advanced Modding Guide**, and how **super long marathons can now be ranked**!
+
+### The osu! Map Cast
+
+Mentioned in [last month's Mappers' Report](https://osu.ppy.sh/home/news/2020-04-11-mappers-report-march), the **osu! Map Cast** is still going strong, with five new episodes! Usually once per week, this podcast hosted by [Cinnabun](https://osu.ppy.sh/users/11868659) features him talking with a wide variety of mapping community members about osu! and beyond. In the past month and a bit, the osu! Map Cast has published new episodes with [the Elite Nominator Cheri](https://www.youtube.com/watch?v=KpuEXzbmrys), [the osu!taiko Nominator incandescence](https://www.youtube.com/watch?v=poaLJGGeD68), [osu! NAT and former video mapping tutorial extraordinaire pishifat](https://www.youtube.com/watch?v=pJwSShQT3eA), [the new osu! Beatmap Nominator wafer](https://www.youtube.com/watch?v=91oq-hgHAdY), and [a group episode with Nevo, Cheri, incandsecence and Cinnabun all together!](https://www.youtube.com/watch?v=J8Wc8yX4ems) The podcast episodes cover a wide variety of topics, both within osu! such as mapping and modding experiences, and outside of osu! such as birds. Be sure to check out more details and links on the [osu! Map Cast forum post](https://osu.ppy.sh/community/forums/topics/1043221) if this so far has spiked your interest!
+
+### Electoz Advanced Modding Guide
+
+Making waves in the modding community recently is the freshly published modding guide by [Electoz](https://osu.ppy.sh/users/6485263) who is a former Beatmap Nominator and long-time mentor in the Mapping Mentorship Program. The modding guide which can be found on their userpage goes in-depth on basically everything you would need to know about modding at a more advanced level. Electoz has fully infused the guide with his own knowledge and experiences, and the guide itself is largely based off how they would mentor successful aspiring modders! If you're into modding, or want to get into it, checking out the Electoz Advanced Modding Guide should definitely be worth your while.
```

Comment:
Electoz mentioned on twitter that he'd be updating it semi-frequently, so if we did a direct link it'd end up outdated, compared to telling people where to find it which would always be the most recent version.  If changed to add the link I'd do this

```suggestion
Making waves in the modding community recently is the freshly published modding guide by [Electoz](https://osu.ppy.sh/users/6485263) who is a former Beatmap Nominator and long-time mentor in the Mapping Mentorship Program. The modding guide which can be found on their userpage and [here](https://electoz.s-ul.eu/N7Y53Jaj) goes in-depth on basically everything you would need to know about modding at a more advanced level. Electoz has fully infused the guide with his own knowledge and experiences, and the guide itself is largely based off how they would mentor successful aspiring modders! If you're into modding, or want to get into it, checking out the Electoz Advanced Modding Guide should definitely be worth your while.
```